### DISCUSSION TITLE
New trademark policy

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#22785] [General]
Hey all,

Just came across this ( https://joinmastodon.org/trademark ) and appears to prevent the use of the word "mastodon" or the logo in any image or domain name, specifically this part "You may not use the Mastodon word mark, or any similar mark, in your domain name, unless you have written permission from Mastodon gGmbH."

So I'm assuming anyone with the name in their domain is now in breach of this?

If this is the way it's going then doesn't give me much faith in continuing to operate an instance.

**Comment by rbairwell** [comment]
Gargron, the main developet of Mastodon and owner of Mastodon gGmbh has said in the Discord:
> One thing of note is that basically it's not possible (or at least, legally advisable) to give blank permission to use "mastodon" in domain names, *however* it can be covered by giving explicit written permissions and I intend to support everyone who currently has a "mastodon" domain name with a written permission.

and

> I wanted to grandfather-in as much existing behaviour as possible

If you have any queries, it'll be worth reaching out to him via the legal/trademark email address given on the page. I doubt he'll say "no" to most existing things, but giving an affirmative "yes" to how you want to use the Mastodon trademark (which has actually been registered for some time now) is something only he and/or his legal team can do.

**Comment by Gargron** [comment]
The trademark policy covers use cases that *don't require a written permission* (and, for the sake of clarity, that extend beyond fair and nominative use for which you do not need to abide by any policy). It does not mean you can't run a Mastodon server with "mastodon" in the domain name, but you do need a written permission to do it, which I will happily provide for all current domains.

My legal counsel can be my witnesses that I wanted to create a policy that grandfathers-in as much of what currently exists as possible implicitly (i.e. without requiring explicit written permission), but was told that it was not a legally viable strategy. One thing to note is that the trademark has already existed with or without this policy, what this policy does is give you *more* legal reassurance than you had before for cases that are covered therein.

As the policy says, simply reach out to <trademark@joinmastodon.org> if you're affected. As for my part, I intend to reach out to the affected domains from my side as well in the coming days/weeks.

**Reply by sepTN** [reply]
I'm setting up a community Mastodon site for my country, the domain itself won't have "Mastodon", but I'd be using that term in our websites, as well as mastodon related assets/images. Should I reach that email as well to request usage permission for legal assurance? Legality stuffs aren't exactly my forte

**Reply by rbairwell** [reply]
Personally, I would think that would fall under the "[General Guidelines](https://joinmastodon.org/trademark)" section @septN . "Only use the Mastodon marks to accurately identify those goods or services that are built using the Mastodon software" (as long as you keep running Mastodon and don't switch to Pleorma for example) and "You may use the Mastodon marks when embedding or otherwise displaying user generated content published using the Mastodon software." - and also under the Server Guidelines "If you run your own Mastodon server using the Mastodon software, ...As long as you abide by the terms and conditions of this agreement, you may use the Mastodon marks included in the Mastodon server software for the purposes of running the server."

However, I'm not a lawyer and neither is Gargron. Whilst you can email him at trademarks address he will probably only be able to give general guidance that he's received from his legal advisors (such as the grandfathering in of existing domains). If you want 100% legal certainty, you'll have to get a lawyer familiar with trademark law in your country/region and get them to contact Gargrons.

I doubt he'll send legal letters as long as you are use the trademarks in "fair" usage/"good faith" - but if you were to start advertising your server as "Official Mastodon Server"/"Join Mastodon v2 for $10/pm" or things like that, then I  would expect you will stand a very high chance of lawyers writing to you. But again, I'm not a lawyer and this isn't legal advice.

**Comment by pinobatch** [comment]
Twitter, another well-known microblog platform, publishes [brand guidelines](https://about.twitter.com/en/who-we-are/brand-toolkit) that allow "locking up" the bird logo with a hashtag or user handle. The guidelines recommend these, from left to right:

1. Logo scaled to 100% of the height of the text
2. Space roughly 1/4 of the width of the logo
3. `#` for a hashtag or `@` for a handle
4. The hashtag or handle

Is there a chance of getting a similar construction approved for the Mastodon logo? Or is there a generic fediverse icon that should be used instead for handle lockups?

**Reply by mailbun** [reply]
While it's very late of a reply, there exists a generic fediverse logo commonly used in community, and it's marked CC0 and doesn't seem to be a part of restrictive trademark as of right now - https://commons.wikimedia.org/wiki/File:Fediverse_logo_proposal.svg

**Comment by mailbun** [comment]
For the sake of informing people about context, I will link some newer related information:

Pull https://github.com/mastodon/mastodon/pull/38548 - Merged pull request that adds warning about trademark policy to setup.

Issue https://github.com/mastodon/mastodon/issues/20771 - Newer comments contain some discussion about the above pull request, mine included.
